src/components/RichTextEditor/TextStyleDropdown/ListBox.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import { AriaListBoxOptions, useListBox, useListBoxSection, useOption } from "@react-aria/listbox";
+import { ListState } from "@react-stately/list";
+import type { Node } from "@react-types/shared";
+import * as React from "react";
+
+interface ListBoxProps extends AriaListBoxOptions<unknown> {
+    listBoxRef?: React.RefObject<HTMLUListElement>;
```

Comment:
I'd suggest you import the RefObject directly
```suggestion
    listBoxRef?: RefObject<HTMLUListElement>;
```